signac_dashboard/templates/layout.html

```diff
@@ -99,9 +99,17 @@ <h2 class="subtitle is-4">Modules</h2>
                         </a>
                         {% endif %}
                         {% endblock module_selector %}
-                        {% endblock sidebar %}
-                        {% endif %}
                     </div>
+                    {% block logout_link %}
+                    <div class="menu">
```

Comment:
I added a logout link to the sidebar, but I need help getting it to appear at the bottom of the sidebar instead of underneath the view and module selection boxes.

Reply:
I'm not familiar with the css theme that dashboard uses. Does it not have an existing class for this? If not, you'll need to add css code. This is the first hit in a search: https://css-tricks.com/float-an-element-to-the-bottom-corner/

However, consider adding the log out link to the upper right (can use css `float: right;`). That is where I expect to find it on the majority of web pages.